packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -408,12 +408,11 @@ const VirtualListBaseFactory = (type) => {
 				{dimensionToExtent, primary} = this.uiRef,
 				{findSpottableItem} = this,
 				{firstVisibleIndex, lastVisibleIndex} = this.uiRef.moreInfo,
-				numOfItemsInPage = (Math.floor((primary.clientSize + spacing) / primary.gridSize) * dimensionToExtent),
-				isPageDown = (direction === 'down' || direction === 'right') ? 1 : -1;
```

Comment:
This kind of code left in `scrollToNextItem` too. Please clean them up.

Reply:
I cleaned up.